tests/update/updates_packagekit_kde.pm

```diff
@@ -24,7 +24,9 @@ sub kernel_updated {
 
 sub turn_off_screensaver() {
     x11_start_program("kcmshell5 screenlocker");
-    send_key("alt-l");
+    if (check_screen 'kde-screenlock-enabled') {
+        assert_and_click('kde-disable-screenlock');
+    }
     assert_screen 'screenlock-disabled';
```

Comment:
Generally I try to avoid a check_screen with default timeout or longer. In your approach, if !enabled, it will wait for 30 seconds until the check_screen "fails" and then assert that it actually is disabled. asserting on multiple tags is mor resembling what a user would do: Expect any alternative, act accordingly. You get the same screenshot anyway in case none match but we save 30 seconds in every test run where screenlock is already disabled. I replaced a lot of occurences already to save time and also to fix problems. I also tried to add a helper function which would make that more convenient to the test writers but so far [I could not pass coolo/aaannz-gate](https://github.com/os-autoinst/os-autoinst/pull/546/files) ;-)